Main.sublime-menu

```diff
@@ -0,0 +1,79 @@
+[
+    {
+        "id": "preferences",
+        "children":
+        [
+            {
+                "id": "package-settings",
+                "children":
+                [
+                    {
+                        "caption": "SublimeKickAssemblerC64",
```

Comment:
I included settings/preferences in menu for SublimeKickAssemblerC64 in master, but using, the `edit_settings`-command. You need to resolve some conflicts, sorry for that.

Reply:
Sorry for being out :) I have to learn the github workflow now :)

Reply:
ok. I think i resolved the conflict now.